### PR TITLE
Markdown: Resume, Presentation.

### PR DESCRIPTION
**Pull request by ashutosh2411**
Templates for Resume and Presentation added. 
Instructions and tools to generate PDFs and Presentations included in README.
ToDo: templates for Reports.

### CODE REVIEW

**Comment by piyush-kurur** [comment]
For each template in format `foo` I have followed the convention that all the relevant files are kept
in the directory `foo/name_of_the_template/` (note it is a directory and not a file). This way any one who wants to work on a document can copy the directory,  put it under version control, and start editing. This directory is also the right place to put a readme on any special instructions needed for
the format. 

Please follow that convention.

Also there is no point in adding the official template from a project; you can as well point to that. Rather
what you would want are tweaks specific to IIT Palakkad. For example, there is no point in keeping
a `reports.cls` file in this repository. However, it might make sense to configure it appropriately so that
a student can use the latex files there (together with supporting files like a make file and a set of custom macros).

File: markdown/resume.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+* ___(Year)___ Received a particular honour.
+
+## Technical Skills
+
+* ___Advanced___ __:__ C, Java, Python, MATLAB, LATEX
+* ___Beginner___ __:__ C++, Haskell, HTML, CSS
+
+## Projects
+
+* __Project 1:__ _(Jan, 2018 - Feb, 2018)_ Designed and implemented a Hashing protocol in Haskell for encrypting and decrypting messages on LAN. Find the implementation [here](github.com).
```
Comment (piyush-kurur) [inline markdown/resume.md]: I can see that this is one of your projects. I would prefer a fictitious project (some joke project is better).

File: markdown/resume.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+* __Project 2:__ _(March, 2018 - April, 2018)_ Similar short description.
+
+## Professional Experiences
+
+* __Research Intern @IIT Madras:__ _(May 2017 - July 2017)_ Worked on some important stuff. Certificate could be found [here](drive.google.com).
+* __Software Developer Intern @AgniCool:__ _(May 2018- July 2018)_ Worked in agile environment. [Here](google.com) is a link to Google's home page.
```
Comment (piyush-kurur) [inline markdown/resume.md]: Same here, Joke projects are encouraged.

**Review by piyush-kurur — CHANGES_REQUESTED** [review]

**Comment by ashutosh2411** [comment]
The repository structure has been modified 
The slides template has been tweaked
Internship reports and project reports have been updated.

**Comment by piyush-kurur** [comment]
I will merge this at some point of time.In response to review message you just need to keep pushing on to the same branch. You do not have to close.

**Pull request closed without merging**